### ISSUE TITLE
Text display issue: Energy Crystal, Military Armament & Discordian Visor miracle upgrade description

### ISSUE THREAD

**Issue by Fluff8836** [DoukutsuRandamu/IssueTracker#80]
One empty line at the end of the description, shown in this screenshot below.
Start.tsc, "for any weapon to not consume ammo." is at 35 characters in one line.
```
#0607
<PRI<MSG<GIT1014Energy Crystal hastens
recharge of Machine Gun and Spur.
and allows Polar Star and Nemesis
to recharge slowly.<NOD It also 
increases max ammo on Polar Star
and Nemesis and provides chance
for any weapon to not consume ammo.<NOD<CLRCraft it?<YNJ0311<CLR<GIT0000.<WAI0020.<WAI0020.<WAI0020.<WAI0060<CLR<CMU0010<GIT1014<IT+0014<IT-0020<IT-0045<FL+0861<EQ+P776<MM-Created =Energy Crystal=!<WAI0135<NOD<CLO<PS-0007<RMU<END
```
One empty line in the middle of description, shown in the screenshot below.
Start.tsc, "40 seconds cooldown. It also shoots" is at 35 characters in one line.

```
#0621
<PRI<MSG<GIT1073Military Armament uses
nanomachines to protect <NAM
from damage.<NOD This ability has
40 seconds cooldown.<NOD It also shoots
homing projectiles on attacking
enemies<NOD and
semi-frequently from
nanites shield.<NOD<CLRCraft it?<YNJ0311<CLR<GIT0000.<WAI0020.<WAI0020.<WAI0020.<WAI0060<CLR<CMU0010<GIT1073<IT+0073<IT-0072<IT-0055<FL-0712<E2-0029<FL-0882<MM-Created =Military Armament=!<WAI0135<NOD<CLO<PS-0021<RMU<END
```
Versions 1.44.0.0 and prior.

![image](https://github.com/DoukutsuRandamu/IssueTracker/assets/153852440/17b09aec-6db0-4e53-afa0-a2c0384a752c)
![image](https://github.com/DoukutsuRandamu/IssueTracker/assets/153852440/abd06f7a-0e95-4549-9e84-8f677584083b)

**Comment by Fluff8836** [comment]
Adding something more...

One empty line in the middle of description, shown in the screenshot below.
Start.tsc, "20 seconds cooldown. It also grants" is at 35 characters in one line.

```
#0613
<PRI<MSG<GIT1026Discordian Visor increases
weapon damage by 25% and allows
to teleport with mouse. This has
20 seconds cooldown.<NOD It also grants
special teleportation dash
which goes through walls.<NOD<CLRCraft it?<YNJ0311<CLR<GIT0000.<WAI0020.<WAI0020.<WAI0020.<WAI0060<CLR<CMU0010<GIT1026<IT+0026<IT-0053<IT-0058<FL-0703<E2-0024<FL-0737<MM-Created =Discordian Visor=!<WAI0135<NOD<CLO<PS-0013<RMU<END
```
![image](https://github.com/DoukutsuRandamu/IssueTracker/assets/153852440/8dd7c13f-3ff4-46e4-9249-011be88fdd8e)

**Comment by TrashboxBobylev** [comment]
Fixed in 1.44.1.

**Comment by Fluff8836** [comment]
Only corrected event 607. But did not correct events 613 and 621 mentioned. Start.tsc.

Version 1.44.1.1.

**Comment by TrashboxBobylev** [comment]
Fixed events 613 and 621 in 1.44.2.
